clang/test/dpct/helper_files_ref/include/sparse_utils.hpp

```diff
@@ -262,6 +262,524 @@ void optimize_csrsv(sycl::queue &queue, oneapi::mkl::transpose trans,
 }
 #endif
 
+/// Structure for describe a dense vector
```

Comment:
Can you explain the difference between this file and the sparse_utils.hpp.inc file?  it sort of looks like one has event handling and the other doesn't.  Is that it?  I put all my comments in the sparse_utils.hpp.inc file, but most apply to both files.

Reply:
sparse_utils.hpp.inc the "source" code, before building dpct, there is a python script generating some special info about the helper functions and a final header file (clang/test/dpct/helper_files_ref/include/sparse_utils.hpp is the reference used to check the header final file).
The only difference of those two files is that *.hpp.inc file conatins special comment like "// DPCT_???" which will be parsed by a python script.
We plan to remove those  *.hpp.inc file recently, so you can treat those two files are same.